package.json

```diff
@@ -32,6 +32,7 @@
     "docs-test": "linkinator docs",
     "fix": "gts fix",
     "lint": "gts check",
+    "prefix": "cd samples/ && npm link ../ && npm i",
```

Comment:
Adding this as a `pre` to the `fix` command means every time someone runs `npm run fix`, it's going to go through the steps of cding into samples, doing the link, and an npm install.  That's going to be *hella* slow, and I'm guessing not entirely what we want.  If the goal is to make sure we take these steps in a CI environment, adding this:
```js
"prelint": "cd samples; npm link ../; npm install",
```

May be a better step.  Generally folks don't run `npm run lint` locally, and tend to run `npm run fix` locally.  I'm far happier to make this change in the CI environment rather than where it affects the local developer flow.

Reply:
Okay, I realized that the prelint script that was present was `"prelint": "cd samples/ && npm link ../ && npm i"` and I'm wondering if the difference in this script and the one you suggested is what's causing the failure in the CI environment.